openc3-cosmos-init/plugins/openc3-tool-base/public/js/auth.js

```diff
@@ -35,9 +35,14 @@ class Auth {
     delete localStorage.openc3Token
   }
   login(redirect) {
+    let url = new URL(redirect)
+    let result = url.pathname
+    if (url.search) {
+      result = result + url.search
+    }
```

Comment:
What are you accomplishing here? Comment?

Reply:
This converts to a relative url.